src/components/Tinder/Tinder.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+import getStore from './../../utils/honestyStore';
+import {ControllerDataset} from '../Admin/Trainer/ControllerDataset';
+
+class Tinder extends Component {
+  state = {
+    loading: true,
+    image: null
+  };
+
+  nextImage = async () => {
+    return this.controllerDataset
+      .getUntrustedImage()
+      .then(image => this.setState({image}));
+  };
+
+  trustImage = () => {
+    this.controllerDataset
+      .trustImage({id: this.state.image.id, item: this.state.image.label})
+      .then(this.nextImage);
+  };
+
+  setAsUnknown = () => {
+    this.controllerDataset
+      .setLabel(this.state.image.id, 'unknown')
+      .then(this.nextImage);
+  };
+
+  deleteImage = () => {
+    this.controllerDataset
+      .deleteImage({id: this.state.image.id, item: this.state.image.label})
+      .then(this.nextImage);
+  };
+
+  changeCategory = event => {
+    this.controllerDataset
+      .setLabel(
+        this.state.image.id,
+        Object.values(this.storeList).find(
+          item => item.name === event.target.value
+        ).id
+      )
+      .then(() =>
+        this.setState(
+          {
+            showDropdown: false
+          },
+          this.nextImage
+        )
+      );
+  };
+
+  componentDidMount() {
+    getStore()
+      .then(storeList => {
+        storeList['unknown'] = {name: 'unknown'};
+        this.storeList = storeList;
+      })
+      .then(() => {
```

Comment:
We don't need the other then, it's not hurting anything but since we're not waiting for anything sync/returning anything it's not adding much

e.g.

```
.then(storeList => {
  this.storeList = {
    ...storeList,
    unknown: {
      name: 'unknown',
    }
  }

  this.nextImage().then(() => {
    this.setState({ loading: false })
  })
})
```

Reply:
I did it this way because `render()` uses storeList to display the item name. If storeList isn't loaded first `render()` will crash